src/subpages/list.tsx

```diff
@@ -1,263 +1,138 @@
-    },
-    {
-      key: 'columnHash',
-      name: 'Block Hash',
-      fieldName: 'hash',
-      minWidth: 5,
-      maxWidth: 450,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: false,
-      data: 'string',
-      isPadded: true,
-      onRender: ({ hash }) => <Link href={`./block/?${hash}`}>{hash}</Link>,
-    },
-    {
-      key: 'columnTimestamp',
-      name: 'Timestamp',
-      fieldName: 'timestamp',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'string',
-      isPadded: true,
-      onRender: ({ timestamp }) => <Timestamp timestamp={timestamp} />,
-    },
-    {
-      key: 'coulmnMiner',
-      name: 'Miner',
-      fieldName: 'miner',
-      minWidth: 123,
-      maxWidth: 450,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'string',
-      isPadded: true,
-      onRender: ({ miner }) => (
-        <Link href={`./account/?${miner}`}>{miner}</Link>
-      ),
-    },
-    {
-      key: 'columnTimeTaken',
-      name: 'Time Taken',
-      minWidth: 50,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'string',
-      isPadded: true,
-      onRender: (block, index) => {
-        let beforeBlock = blocks[Math.min(index! + 1, blocks.length - 1)];
-        let beforeTimestamp = Date.parse(beforeBlock.timestamp);
-        let nowTimestamp = Date.parse(block.timestamp);
-        return <>{(nowTimestamp - beforeTimestamp) / 1000}</>;
-      },
-    },
-    {
-      key: 'columnDifficulty',
-      name: 'Difficulty',
-      minWidth: 50,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'string',
-      isPadded: true,
-      onRender: ({ difficulty }) => (
-        <>{parseInt(difficulty).toLocaleString()}</>
-      ),
-    },
-    {
-      key: 'columnTxNumber',
-      name: 'Tx #',
-      minWidth: 5,
-      maxWidth: 20,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: false,
-      data: 'number',
-      isPadded: true,
-      onRender: ({ transactions }) => <>{transactions.length}</>,
-    },
-  ];
+const SummaryCards: React.FC<SummaryCardsProps> = ({ blocks }) => {
+  if (blocks === null)
+    return <Cards interval={0} difficulty={0} totalTxNumber={0} />;
+
+  const timestamps: Date[] = blocks.map(block => new Date(block.timestamp));
+
+  let interval = 0;
+  for (let i = 0; i < timestamps.length - 1; i++) {
+    interval += +timestamps[i] - +timestamps[i + 1];
+  }
+  interval /= (timestamps.length - 1) * 1000;
+
+  const difficulties = blocks.map(block => block.difficulty);
+  const difficulty =
```

Comment:
I believe It will be better to use another variable name like `difficultyAverage` or better one because it means *average of difficulties of blocks*. Also `interval` too.

Reply:
I agree rename to difficultyAverage. But I think `interval` is more readable.